model_utils.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import print_function
+
+import numpy as np
+from matplotlib import pyplot as plt
+
+
+def prepare_pixelized_dataset(dataset, window_x=5, window_y=5,
+                              y_applied_function=np.asarray):
+    Xs = []
+    ys = []
+    for datum in dataset:
+        img = datum['img']
+        mask = datum['mask']
+        for selection, mask_selection in prepare_pixelized_image(img, mask):
```

Comment:
@mrshu I believe there should be `window_x` and `window_y` passed here to `prepare_pixelized_image` function or be removed from head of this function since it is not used anywhere